src/checkout/checkout-store-error-selector.ts

```diff
@@ -330,7 +332,7 @@ export default class CheckoutStoreErrorSelector {
      *
      * @returns The error object if unable to apply, otherwise undefined.
      */
-    getApplyCouponError(): Error | undefined {
+    getApplyCouponError(): RequestError<StorefrontErrorResponseBody> | undefined {
```

Comment:
Correct me if I'm wrong, but I think these getters return more than just `RequestError`. Any error that's thrown inside an observable action will be surfaced through here. So we might still need to have `Error` or `CustomError`.

Reply:
at the moment this error is generated in the couponActionCreator, when the promise from couponRequestSender fails, so I think it's safe to assume it's RequestError in this case.